python/lsst/ts/vent/controller/controller.py

```diff
@@ -453,42 +469,50 @@ def get_opto_ch(self, stack_number: int, channel_number: int) -> int:
         AssertionError
             If the controller is not connected.
 
-        NotImplementedError
-            If we are not in simulation mode.
+        OSError
+            If there is a communication error.
         """
 
         assert self.connected
         if self.simulator is not None:
-            return self.simulator.get_opto_ch(stack_number, channel_number)
+            return self.simulator.read_channel(bus_number, stack_number, channel_number)
         else:
-            raise NotImplementedError("Sequent hardware not implemented.")
+            sequent.read_channel(bus_number, stack_number, channel_number)
 
-    def set_od(self, stack_number: int, channel_number: int, value: int) -> None:
+    def write_channel(
+        self, bus_number: int, stack_number: int, channel_number: int, value: int
+    ) -> None:
         """Calls harware I/O or a simulated substitute depending
         whether the class was instantiated with simulate = True.
 
         Parameters
         ----------
+        bus_number : int
+            I2C hardware bus number. The hardware is addressable by the
+            device /dev/i2c-{bus_number}.
+
         stack_number : int
             The hardware stack number for the I/O card.
 
-        channel_number: int
+        channel_number : int
             The I/O channel number to write.
 
-        value: int
+        value : int
             The value to send to the I/O output, 1 for high or 0 for low.
 
         Raises
         ------
         AssertionError
             If the controller is not connected.
 
-        NotImplementedError
-            If we are not in simulation mode.
+        OSError
```

Comment:
you are not raising this exception here yourself so, I think you can remove this.